src/main/java/recipeio/CommandValidator.java

```diff
@@ -43,8 +43,8 @@ public static boolean isParsableAsInteger(String input) {
             Integer.parseInt(input);
             return true;
         } catch (NumberFormatException e) {
-            System.out.println("\tParameter cannot be parsed as an integer.");
-            System.out.println("\tPlease enter an integer from 1 onwards.");
+            System.out.println("Parameter cannot be parsed as an integer.");
+            System.out.println("Please enter an integer from 1 onwards.");
```

Comment:
I recall we discussed about indenting the console output. Do you feel no indentation is better?